blog/_posts/2017-08-10-opensource-here-we-come.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Open source, here we come!
+layout: post
+created_at: Monday August 10 2017
```

Comment:
Hi, can you please change the date to September 4, 2017? Change permalink field as well.